### PR TITLE
support component names when they are created as ES6 classes

### PR DESCRIPTION
**Pull request by leonid-shevtsov**
@mikekreeki this fixes the 'undefinedContainer' transmit container name which harms performance analysis a lot.

### CODE REVIEW

**Comment by mikekreeki** [comment]
@XeeD Can you please share your thought on this?

File: src/lib/createContainer.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ module.exports = function (Component, options) {
 	options = arguments[1] || {};
 
 	var Container = React.createClass({
-		displayName: Component.displayName + "Container",
+		displayName: (Component.displayName||Component.name) + "Container",
```
Comment (mikekreeki) [inline src/lib/createContainer.js]: Please, always add space around `||`, `+` etc.

**Comment by XeeD** [comment]
If it works, fine :+1:

**Comment by mikekreeki** [comment]
:+1:

**Comment by XeeD** [comment]
@leonid-shevtsov Could you please open a PR to react-transmit? I think everybody should benefit from this.

**Pull request merged**